### DISCUSSION TITLE
English only labels, directions using pins

### DISCUSSION THREAD

**Discussion by pmsoltani** [FacilMap/facilmap#178] [Q&A] [answered]
First of all, thanks for the great web service!

Now, I have 2 questions:

1. Is there a way to show all labels in English, instead of the native language? Something like [osmap.us](https://www.osmap.us/)
2. On the OSM website, I can find the route between two manually placed pins (picture below). Can I do the same using FacilMap?

![image](https://user-images.githubusercontent.com/25427031/121798903-2d996b80-cc3e-11eb-9d0a-76d536f3a383.png)

Thanks 🙏

**Comment by cdauth** [comment, marked as answer]
To add a route between map points, [long click the map](https://docs.facilmap.org/users/click-marker/) and then use the “Use as” menu.

Right now there seems to be no English-only map layer. TopPlus shows German labels, which might help you already in some places if your problem is that you can only read the Latin alphabet. But in places that don't have a specific German name, it seems to show the native names instead of the English one. I will think about a solution to this.

**Reply by pmsoltani** [reply]
Appreciate the quick reply 🙏

**Comment by cdauth** [comment]
The new default layer is LimaLabs. It will be released with FacilMap 4, but your can already try it out on https://beta.facilmap.org/. It shows English labels in addition to local ones.
